docs/basics/mass_transit_integration.md

```diff
@@ -0,0 +1,60 @@
+# MassTransit Integration
+
+[`LeanCode.CQRS.MassTransitRelay`] allows to pass raised events to [MassTransit](https://masstransit-project.com/) bus. This is the only way of consuming domain events.
```

Comment:
Looks like shorthand reference-style links with code-formatted text doesn't work with the MkDocs, there's no link on the RedTheDocs page, only the literal [`LeanCode.CQRS.MassTransitRelay`]. I think the classic two-part reference links could do the trick:
```markdown
[`LeanCode.CQRS.MassTransitRelay`][LeanCode.CQRS.MassTransitRelay]

[LeanCode.CQRS.MassTransitRelay]: https://github.com/leancodepl/corelibrary/tree/v8.0-preview/src/CQRS/LeanCode.CQRS.MassTransitRelay
```

Reply:
Note: the ``[`AddMassTransit`](https://masstransit.io/documentation/configuration)`` further in text works fine.

Reply:
The problem is, I think, combining links with inline code - "just" link works. Thanks for noting!

Reply:
Should work now